src/tapis-ui/_common/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -29,7 +29,7 @@ const Breadcrumbs: React.FC<BreadcrumbsProps> = ({ breadcrumbs }) => {
   return (
     <div>
       {breadcrumbs.map((item, index) => {
```

Comment:
use `truncatedBreadcrumbs` here, and do not reassign value of breadcrumbs param

Reply:
truncatedBreadcrumbs now set to breadcrumbs and not the other way around